xptifw/samples/include/xpti_timers.hpp

```diff
@@ -0,0 +1,95 @@
+//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+#pragma once
+
+#include <atomic>
+#include <sstream>
+#include <thread>
+#include <unordered_map>
+
+namespace xpti {
+class ThreadID {
+public:
+  typedef std::unordered_map<std::string, int> thread_lut_t;
+
+  ThreadID() : m_tid(0) {}
```

Comment:
Why not removing it and just initialize `m_tid` on line 48?

Reply:
Done!